### PR TITLE
Allow more escalation step wait times over public api

### PR DESCRIPTION
**Pull request by mderynck**
# What this PR does
Remove restriction on wait times when changing escalation step wait times.  60-86400 seconds values are accepted.

https://github.com/grafana/terraform-provider-grafana/pull/1855 removes the restriction in the terraform provider.

## Which issue(s) this PR closes

Related to: https://github.com/grafana/support-escalations/issues/13065

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] Added the relevant release notes label (see labels prefixed w/ `release:`). These labels dictate how your PR will
    show up in the autogenerated release notes.

### CODE REVIEW

File: docs/sources/oncall-api-reference/escalation_policies.md [hunk 1]
```diff
@@ -38,22 +38,22 @@ The above command returns JSON structured in the following way:
...
+| `type`                             |                   Yes                    | One of: `wait`, `notify_persons`, `notify_person_next_each_time`, `notify_on_call_from_schedule`, `notify_user_group`, `trigger_webhook`, `resolve`, `notify_whole_channel`, `notify_if_time_from_to`, `declare_incident`.                                                                                 |
+| `important`                        |                 Optional                 | Default is `false`. Will assign "important" to personal notification rules if `true`. This can be used to distinguish alerts on which you want to be notified immediately by phone. Applicable for types `notify_persons`, `notify_team_members`, `notify_on_call_from_schedule`, and `notify_user_group`. |
+| `duration`                         |            If type = `wait`              | The duration, in seconds, when type `wait` is chosen. Valid values are: `60-86400`.                                                                                                                                                                                                                        |
```
Comment (matiasb) [inline docs/sources/oncall-api-reference/escalation_policies.md:47]: Maybe is not that clear enough that any value in the range is valid?

File: engine/apps/public_api/serializers/escalation_policies.py [hunk 2]
```diff
@@ -143,18 +151,10 @@ def to_representation(self, instance):
         result = super().to_representation(instance)
         result = self._get_field_to_represent(step, result)
         if "duration" in result and result["duration"] is not None:
-            result["duration"] = result["duration"].seconds
+            result["duration"] = int(float(result["duration"]))
```
Comment (matiasb) [inline engine/apps/public_api/serializers/escalation_policies.py:154]: Is this conversion needed? (from what I see in the field definition, the `to_representation` should return the right repr?)

Comment (mderynck) [reply engine/apps/public_api/serializers/escalation_policies.py:154]: Without this the DurationSecondsField returns a float as a string:
```
{
      "position": 0,
      "id": "E6J46FYHBXGUW",
      "escalation_chain_id": "FDWP4BEVEZD9C",
      "type": "wait",
      "duration": "111.0"
    },
```

Comment (mderynck) [reply engine/apps/public_api/serializers/escalation_policies.py:154]: Probably don't really need to use that DurationSecondsField but I used it to match the other internal API serializer.

**Pull request merged**